src/app/pages/send/send-crypto-asset-form/form/btc/btc-send-form-confirmation-details.tsx

```diff
@@ -33,7 +34,7 @@ export function BtcSendFormConfirmationDetails(props: BtcSendFormConfirmationDet
   return (
     <ConfirmationDetailsLayout amount={amount}>
       <ConfirmationDetail detail="Token" value="Bitcoin" />
-      <ConfirmationDetail detail="To" value={recipient} />
+      <ConfirmationDetail detail="To" value={truncateMiddle(recipient, 4)} />
```

Comment:
Not so sure we want to do this. Users should be able to verify they're sending to the correct address.

cc/ @mica000 should consider this in designs as address can be long. Maybe we can reuse this component some how 

<img width="510" alt="image" src="https://user-images.githubusercontent.com/1618764/219571166-a7cfa91e-eb9a-4852-9a93-2c6711c51c68.png">

Reply:
https://bitcoin.design/guide/glossary/address/#visual-formatting